platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/AbstractStsRealm.java

```diff
@@ -110,6 +113,10 @@
 
     private Boolean useKey = null;
 
+    private Cache<Element, SecurityToken> cache = CacheBuilder.newBuilder()
+            .expireAfterWrite(1, TimeUnit.MINUTES)
```

Comment:
You cannot have a session with a life shorter than 1 minute. The trigger for us to renew a session is at the 1 minute mark.